src/pages/CharacterPage.js

```diff
@@ -0,0 +1,15 @@
+import "../App.css";
+import Header from "../components/Header";
+import DetailedCard from "../components/DetailedCard";
+import NavBar from "../components/NavBar";
+
+export default function CharacterPage() {
+  
+  return (
+    <div className="App">
```

Comment:
You can also work here with a `styled component`instead of an extra `App.css`. :-)

Reply:
Could I please keep it like that for now? :) Thanks!